lib/client/src/health_cache.rs

```diff
@@ -34,6 +34,7 @@ pub async fn new(
         begin_perp: active_token_len * 2,
         begin_serum3: active_token_len * 2 + active_perp_len * 2,
         staleness_slot: None,
+        fallback_oracle_ais: vec![],
```

Comment:
This poses an interesting question: How do clients deal with fallback oracles?

Processes like the liquidator that have no trouble fetching a few more oracle accounts could grab the fallback accounts, compute staleness client side and then pass in up to a configurable number of fallbacks if needed.

It'll need a bunch of work and this function is one of the places that would be affected.

Some code, like the "is this liquidatable?" check, would want to pass in all fallbacks so we never give up too early. Then presumably later code would figure out what fallbacks are actually needed?

None of this has to happen in this PR, but maybe mark this line as needing to be revisited.

Reply:
I haven't given much thought to liquidators yet, what I mostly had in mind was a user who is struggling to place orders. The client could have a bool for including a fallback oracle on the market they are trying to place on for example. Once this is tested I think we could expand how many fallbacks the client provides (I can definitely see use cases where someone would send as many fallbacks as possible)